src/components/AdminPageTable/AdminPageTable.jsx

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable react/jsx-key */
 import React, { useEffect, useMemo } from 'react';
+import { Button, Table, Tr, Td, Tbody, Spinner, Box, Flex, Text } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Box' is defined but never used.